gazebo_ros/worlds/gazebo_ros_properties_demo.world

```diff
@@ -32,6 +36,10 @@
 
       ros2 service call /demo/set_light_properties 'gazebo_msgs/SetLightProperties' '{light_name: "sun", diffuse: {r: 0.8, g: 0.7, b: 0.2, a: 1.0}, attenuation_constant: 0.9, attenuation_linear: 0.01, attenuation_quadratic: 0.001}'
 
+  Or physics properties:
+
+      ros2 service call /demo/set_physics_properties 'gazebo_msgs/SetPhysicsProperties' '{gravity: {x: 0.0, y: 0.0, z: 0.0}}'
```

Comment:
Making this call seems to break physics, the robot breaks apart and nothing else seems to work. I think it's because all the values not set in the message are being set to zero.

I think there's no good way around it in the code because we can't tell whether a value is unset or set to zero. So I think we should encourage the best practice of filling all the message fields when sending a request. Could you add the default value for all other properties to this example request? Thanks!